### PR TITLE
Minor revision of the contacts page

### PR DESCRIPTION
**Pull request by GBKS**
Addresses #787.

- Various minor screen revisions for consistency
- Moved techniques to address unique payment request format UI treatments further down
- Various minor copy tweaks for clarification

I assume there will be feedback about specific payment request formats and whether should or should not appear in some of the UI mock-ups. If we want to get to that level of detail, then we need to truly create a technical spec for this wallet and list out exactly what is supported or not. I don't think that is super necessary in general. If desired, I would tackle this in a another revision.

🪖[Check the preview](https://deploy-preview-812--sad-borg-390916.netlify.app/guide/daily-spending-wallet/contacts/)⛑

### CODE REVIEW

**Review by pavlenex — APPROVED** [review]

File: guide/daily-spending-wallet/contacts.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ imagesSupportedFormats:
       caption: Particular properties of address types can be pointed out right away.
     - file: manual-add-contact-with-on-chain-address
       alt: A contact with an on-chain address assigned
-      caption: A contact with an on-chain address attached. Once a payment is made to the address, it disappears from this list.
+      caption: A contact with an on-chain address attached. Once a payment is made to the address, it disappears from this list as it should only be used once for best privacy.
```
Comment (Bosch-0) [inline guide/daily-spending-wallet/contacts.md:41]: For the images, are we going to link each frame directly to the corresponding frame in the UI kit as Milli suggested?

File: guide/daily-spending-wallet/contacts.md [hunk 2]
```diff
@@ -200,22 +194,22 @@ The contacts screen should offer various features for editing and organization.
 
 ### Importing an address
 
-In this scenario, the user has copied a Lightning address to the clipboard and opened the app. The clipboard is automatically scanned, the app identifies the address and offers the user appropriate options. Here, the user adds the address as a new contact.
+This scenario can be initiated by copying a Lightning address to the clipboard, scanning a QR code, or tapping a payment link (see [payment entry points]({{ '/guide/daily-spending-wallet/sending/#payment-entry-points' | relative_url }})). An address is passed into the application and, where it's recognized and appropriate options are shown to the user. In the example below, the user adds the address as a new contact.
 
 {% include image-gallery.html pages = page.imagesImportAddress %}
```
Comment (Bosch-0) [inline guide/daily-spending-wallet/contacts.md:199]: This first image should state "You have a lightning address on your clipboard" not "You copied a Lightning address." That sounds like the copy action was taken by the user within their DSW app, not externally which is what this section is covering. 

Also I would have two options in the first frame. One being "Add contact" button (which is already there) which automatically opens into a add new contact frame (avoiding going through the contact list frame). Then a "Add to existing contact" button which goes to the contact list frame where they select an existing contact to add the address to. This removes one extra step to take if you are adding a new contact.

File: guide/daily-spending-wallet/contacts.md [hunk 3]
```diff
@@ -200,22 +194,22 @@ The contacts screen should offer various features for editing and organization.
...
+This sequence is similar to the one above. The difference is that a payment request was passed into the application, which contains different data and also includes a specific user action, and therefore requires different user flows. The one below shows how a user has scanned a payment request and assigns a contact to the payment.
 
-Some payment request formats may include an address that can [receive daily-spending-wallet repeatedly]({{ '/guide/daily-spending-wallet/requesting/#reusable-payment-requests' | relative_url }}). In this case, the address is added to the contact for future use. For [single use payment requests]({{ '/guide/daily-spending-wallet/requesting/#single-use-payment-requests' | relative_url }}), only the payment is linked.
+Some payment request formats may include an address that can [receive repeatedly]({{ '/guide/daily-spending-wallet/requesting/#reusable-payment-requests' | relative_url }}). In this case, the address is added to the contact for future use. For [single use payment requests]({{ '/guide/daily-spending-wallet/requesting/#single-use-payment-requests' | relative_url }}), only the payment is linked.
```
Comment (Bosch-0) [inline guide/daily-spending-wallet/contacts.md:205]: To me, this is covered in the Importing an address section above this section - seems unnecessary to mention this again here

File: guide/daily-spending-wallet/contacts.md [hunk 4]
```diff
@@ -200,22 +194,22 @@ The contacts screen should offer various features for editing and organization.
...
 
 Invoices may also contain recipient names. These can be used to suggest the name for a new contact to the user. Names can be spoofed, so they should not be automatically assigned without user approval.
```
Comment (Bosch-0) [inline guide/daily-spending-wallet/contacts.md:207]:
```suggestion
Payment requests may also contain recipient names. These can be used to suggest the name for a new contact to the user. Names can be spoofed, so they should not be automatically assigned without user approval.
```

Regarding my comment above, referring to them as a payment request makes more sense and aligns more with the requesting page. 

Either way we shouldn't be jumping between calling it a simple payment request and an invoice, which is done in this section.

**Review by Bosch-0 — CHANGES_REQUESTED** [review]
Looking good, left a few suggestions

File: guide/daily-spending-wallet/contacts.md [hunk 5]
```diff
@@ -200,22 +194,22 @@ The contacts screen should offer various features for editing and organization.
...
 ### Importing an address
 
-In this scenario, the user has copied a Lightning address to the clipboard and opened the app. The clipboard is automatically scanned, the app identifies the address and offers the user appropriate options. Here, the user adds the address as a new contact.
+This scenario can be initiated by copying a Lightning address to the clipboard, scanning a QR code, or tapping a payment link (see [payment entry points]({{ '/guide/daily-spending-wallet/sending/#payment-entry-points' | relative_url }})). An address is passed into the application and, where it's recognized and appropriate options are shown to the user. In the example below, the user adds the address as a new contact.
 
 {% include image-gallery.html pages = page.imagesImportAddress %}
 
-This flow can also be initiated by scanning a QR code or tapping a payment link. For more, see [Sending bitcoin &middot; Payment entry points]({{ '/guide/daily-spending-wallet/sending/#payment-entry-points' | relative_url }}).
-
 ### Importing an invoice
```
Comment (Bosch-0) [inline guide/daily-spending-wallet/contacts.md:201]:
```suggestion
### Importing a payment request
```

Payment request seems more fitting and aligns more with the requesting page. You also go back and forth between referring to it as an invoice and payment request in this section - bit confusing

File: guide/daily-spending-wallet/contacts.md (hunk 1 above)
Comment (GBKS) [reply guide/daily-spending-wallet/contacts.md:41]: The UI kit is not set up for this. We'll have to deal with this as its own project at some point. Without thinking this through and having a proper process, we'll end up with tons of broken links quickly.

Comment (Bosch-0) [reply guide/daily-spending-wallet/contacts.md:41]: Alright we'll tackle that later then

**Review by sbddesign — APPROVED** [review]
Yeah, let's discuss the concept of promoting/demoting certain formats once we have gotten further in outline the specs of the DSW.

**Review by Bosch-0 — APPROVED** [review]

**Pull request merged**